src/OpenLoco/src/Audio/Audio.cpp

```diff
@@ -54,11 +52,10 @@ namespace OpenLoco::Audio
 
     [[maybe_unused]] constexpr int32_t kNumSoundChannels = 16;
 
-    static loco_global<uint32_t, 0x0050D1EC> _audioInitialised;
-    static loco_global<bool, 0x0050D554> _audioIsPaused;
-    static loco_global<bool, 0x0050D555> _audioIsEnabled;
-    // 0x0050D5B0
-    static std::optional<PathId> _chosenAmbientNoisePathId = std::nullopt;
+    static uint32_t _audioInitialised;                                     // 0x0050D1EC
```

Comment:
Shouldn't this be also a bool?

Reply:
Yes, no reason for it not to be. I'll change it.